builder-base/versions.yaml

```diff
@@ -3,11 +3,12 @@ ANSIBLE_VERSION: 2.10.0
 BUILDKIT_VERSION: v0.10.5
 DOCKER_BUILDX_VERSION: v0.9.1
 GITHUB_CLI_VERSION: 2.21.1
-GOLANG_VERSION_115: 1.15.15-2
-GOLANG_VERSION_116: 1.16.15-3
-GOLANG_VERSION_117: 1.17.13-2
-GOLANG_VERSION_118: 1.18.10-1
-GOLANG_VERSION_119: 1.19.5-1
+GOLANG_VERSION_115: 1.15.15-3
+GOLANG_VERSION_116: 1.16.15-4
+GOLANG_VERSION_117: 1.17.13-3
+GOLANG_VERSION_118: 1.18.10-2
+GOLANG_VERSION_119: 1.19.5-2
+GOLANG_VERSION_120: 1.20.0-1
```

Comment:
not for this PR, but just an FYI, we'll want to update the Builder Base to use this version as well -- there are instructions in the runbook for a new go version addition that we'll have to follow